proto/aserto/directory/model/v3/model.proto

```diff
@@ -182,104 +104,13 @@ message SetManifestResponse {
-                expression: "this.matches('^[a-z][a-z0-9_-]{1,62}[a-z0-9]$')"
-            }
-            string: {
-                max_len: 64
-            }
-        },
-        (google.api.field_behavior) = REQUIRED
-    ];
-    // manifest version (semver notation)
-    string version                                              = 2
-    [
-        (buf.validate.field) = {
-            required: true,
-            cel: {
-                id: "delete_manifest.version"
-                message: "must be 1, 2 or 3 digits separated by a dot like, 1, 1.2 or 1.2.3"
-                expression: "this.matches('^([0-9]{1,3})(.[0-9]{1,3}){0,2}$')"
-            }
-            string: {
-                max_len: 11
-            }
-        },
-        (google.api.field_behavior) = REQUIRED
-    ];
-}
-
-message DeleteManifestResponse {
-    // empty result
-    google.protobuf.Empty result                                = 1;
-}
-
-message ListManifestsRequest {
-    // pagination request
-    aserto.directory.common.v3.PaginationRequest page           = 9
-    [
-        (google.api.field_behavior) = OPTIONAL
-    ];
-}
-
-message ListManifestsResponse {
-    // manifests metadata
-    repeated Metadata manifests                                 = 1;
-    // pagination response
-    aserto.directory.common.v3.PaginationResponse page          = 9;
-}
 
 message Metadata {
-    // manifest name (unique, lc-string)
-    string  name                                                = 1
-    [
-        (buf.validate.field) = {
-            required: true
-            cel: {
-                id: "get_manifest.name"
-                message: "must be all lowercase, start with a letter, can contain letters, digits, underscores, and dashes, and must end with a letter or digit"
-                expression: "this.matches('^[a-z][a-z0-9_-]{1,62}[a-z0-9]$')"
-            }
-            string: {
-                max_len: 64
-            }
-        },
-        (google.api.field_behavior) = REQUIRED
-    ];
-    // manifest version (semver notation)
-    string  version                                             = 2
-    [
-        (buf.validate.field) = {
-            ignore_empty: true,
-            cel: {
-                id: "list_manifest.version"
-                message: "must be 1, 2 or 3 digits separated by a dot like, 1, 1.2 or 1.2.3"
-                expression: "this.matches('^([0-9]{1,3})(.[0-9]{1,3}){0,2}$')"
-            }
-            string: {
-                max_len: 11
-            }
-        },
-        (google.api.field_behavior) = OPTIONAL
-    ];
     // created at timestamp (UTC)
     google.protobuf.Timestamp created_at                        = 20
     [
         (google.api.field_behavior) = OUTPUT_ONLY
     ];
-    // last updated timestamp (UTC)
-    google.protobuf.Timestamp updated_at                        = 21
```

Comment:
I think it semantically makes more sense to keep UpdatedAt to reflect the last write of the manifest then CreatedAt